contrib/bentoml/README.md

```diff
@@ -1,193 +1,86 @@
-# BentoML Yatai Stack
+# BentoML on Kubeflow
 
-[BentoML Yatai Stack](https://github.com/bentoml/yatai-deployment) is a series of components for deploying models/bentos to Kubernetes at scale
+Starting with the release of Kubeflow 1.7, BentoML provides a native integration with Kubeflow through [Yatai](https://github.com/bentoml/yatai-deployment). This integration allows you to package models trained in Kubeflow Notebooks or Pipelines as [Bentos](https://docs.bentoml.org/en/latest/concepts/bento.html), and deploy them as microservices in a Kubernetes cluster through BentoML's cloud native components and custom resource definitions (CRDs). This documentation provides a comprehensive guide on how to use BentoML and Kubeflow together to streamline the process of deploying models at scale.
 
 ## Requirements
 
-* Kubernetes 1.20 - 1.24
+* Kubernetes 1.20 - 1.25
```

Comment:
Could you also update https://github.com/kubeflow/manifests/blob/master/.github/workflows/bentoml_kind_test.yaml to ensure it's tested with 1.25 cluster?

Reply:
Is there an example that has already been updated to 1.25, which I can follow?